.vuepress/config.js

```diff
@@ -21,31 +21,31 @@ module.exports = {
       nav: [
         {
           children: [
-            {
-              title: "Intro",
-              path: "/intro",
-              directory: true,
-            },
             {
               title: "Getting Started",
               path: "/getting-started",
               directory: true,
             },
             {
-              title: "Name Service",
-              path: "/name-service",
+              title: "Architecture",
+              path: "/architecture",
               directory: true,
             },
             {
               title: "Testnets",
               path: "/testnets",
               directory: true,
             },
+          ]
+        },
+        {
+          title: "Tutorials",
+          children: [
             {
-              title: "Production Tooling",
```

Comment:
This is removed?

I guess it was not really maintained, but it would be good to put this info somewhere

Reply:
It is moved into getting started. This section is a very good practice and I thought It should be included into `Essentials`. I moved into `Getting Started`